app/lib/etl.js

```diff
@@ -88,7 +92,8 @@ function start() {
   });
 }
 
-function safeStart() {
+function safeStart(callback) {
+  completeCallback = callback;
```

Comment:
I needed to add a callback for testing purposes, I couldn't find an elegant way to signal the end of the ETL.

Reply:
I wonder if the tests could await the result instead? http://mochajs.org/#using-async--await